daisy/step_wait_for_instances_signal.go

```diff
@@ -86,6 +87,9 @@ func waitForSerialOutput(s *Step, project, zone, name string, so *SerialOutput,
 	if so.FailureMatch != "" {
 		msg += fmt.Sprintf(", FailureMatch: %q", so.FailureMatch)
 	}
+	if len(so.FailureMatches) > 0 {
```

Comment:
I prefer to let FailureMatch and FailureMatches be mutually exclusive

Reply:
Could be either, I don't see a strong reason to go one way or the other. This was simpler to implement, simpler validation and easier on the user in case they mess up and specify both. I doubt anyone will use both but if they do, it will still work.

Reply:
To me it's confusion to have both accepted. But you can decide based on your best judgement.

Reply:
This is now deprecated after the refactoring to support both arrays and single strings in the old (FailureMatch) field.